errs/errs.go

```diff
@@ -0,0 +1,86 @@
+// Package errs exports New and Errorf, extending the errors package
```

Comment:
Now this is being published and will have godocs available publicly, I'm going to make lots of suggestions for wording of godocs. I would like one quality of foxygoat packages to be that the documentation is really good.

```
// Package errs provides error wrapping functions that allow an error to
// wrap multiple other errors. The error wrapping in the Go standard
// library allows an error to wrap only one error with only one %w
// format verb in the format string passed to fmt.Errorf and the
// errors.Unwrap() function that returns only a single error.
//
// The error type returned by the functions in this package wraps
// multiple errors so that errors.Is() and errors.As() can be used to 
// query if that error is any one of the wrapped errors. errors.Unwrap()
// on errors of that type always returns nil as that function cannot
// return multiple errors.
//
// The error type is not exported so is used through the standard Go
// error interfaces.
```

Reply:
hmm, on the verbose side i'd say. It'll be nice to see what it looks like published.